src/solvers/parabolic_solvers.jl

```diff
@@ -0,0 +1,35 @@
+"""
+  struct BR1 end
+
+The Bassi-Rebay flux from "A High-Order Accurate Discontinuous Finite Element Method for
+the Numerical Solution of the Compressible Navier-Stokes Equations" by Bassi, Rebay (1997).
+
+https://doi.org/10.1006/jcph.1996.5572
+"""
+struct BR1 end
+
+# no penalization for a BR1 parabolic solver
+function calc_viscous_penalty!(scalar_flux_face_values, u_face_values, t, boundary_conditions,
+                               mesh, equations::AbstractEquationsParabolic, dg::DGMulti,
+                               dg_parabolic::BR1, cache, cache_parabolic)
+  return nothing
+end
+
+"""
+  struct LDG{P}
+    penalty_parameter::P
+  end
+
+The local DG (LDG) flux from "The Local Discontinuous Galerkin Method for Time-Dependent
+Convection-Diffusion Systems" by Cockburn and Shu (1998).
+
+Note that, since this implementation does not involve the parabolic "upwinding" vector,
+the LDG solver is equivalent to BR1 with an LDG-type penalization term.
+
+https://doi.org/10.1137/S0036142997316712
+"""
+struct LDG{P}
+  penalty_parameter::P
+end
+
+default_parabolic_solver() = BR1()
```

Comment:
Right now, I feel like `parabolic_solver` (or `solver_parabolic`) is not consistently named. To me, I would have expected as a parabolic *solver* that some sort of `DGSEM`/`DGMulti`-type object would be passed, which might contain also a specific flux that is used.

To me, the current implementation would fit better if named `flux_parabolic` and `default_parabolic_flux` (although that is not optimal either). I am not an expert on these methods, but IMHO that uniquely qualifies me to be confused by the naming scheme 😬 Maybe @gregorgassner or @andrewwinters5000 have an opinion on this as well?

Reply:
`ViscousFormulationBassyRebay1` is a beast in terms of length, though 😬 I do share @andrewwinters5000's concerns with naming it "flux", thus I don't know the best way out of this. Maybe use `ViscousFormulation` for now but add it to #1148?

Reply:
Done and added to https://github.com/trixi-framework/Trixi.jl/issues/1147